modules/virt-enabling-mac-address-pool-for-namespace-cli.adoc

```diff
@@ -5,15 +5,20 @@
 [id="virt-enabling-mac-address-pool-for-namespace-cli_{context}"]
 = Enabling a MAC address pool for a namespace in the CLI
 
-Enable a MAC address pool for virtual machines in a namespace by applying the `mutatevirtualmachines.kubemacpool.io=allocate` label to the namespace.
+If you disabled KubeMacPool for a namespace and want to re-enable it, remove the `mutatevirtualmachines.kubemacpool.io=ignore` label from the namespace.
+
+[NOTE]
+====
+Earlier versions of {VirtProductName} used the label `mutatevirtualmachines.kubemacpool.io=allocate` to enable KubeMacPool for a namespace. This is still supported but redundant as KubeMacPool is now enabled by default.
+====
 
 .Procedure
 
-* Add the KubeMacPool label to the namespace.
-The following example adds the KubeMacPool label to two namespaces, `<namespace1>` and `<namespace2>`:
+* Remove the KubeMacPool label from the namespace.
```

Comment:
Same as above. Suggestion to reword this as "Remove the `mutatevirtualmachines.kubemacpool.io=ignore` label from the namespace. The following example re-enables KubeMacPool  for two namespaces, `<namespace1>` and `<namespace2>`: